javascript/conditonal-statement.md

```diff
@@ -0,0 +1,259 @@
+# CONDITIONAL STATEMENT IN JAVASCRIPT.
+
+Conditional statements are statements that perform particular tasks based on the given condition(s).
+
+## This project explains the following:
+
+- What conditional statements are.
+- The `if` statement.
+- The `if...else` statement.
+- Logical operators.
+- ternary operators.
+- Truthy and falsy.
+- Truthy and falsy assignment.
+- The `else...if` statement.
+- Switch keyword.
+
+## The `if` statement:
+
+Tasks are mostly carried out based on condition. For example, if you are hungry, then you will eat. If it rains, then you use your umbrella.
+
+The same thing applies to our code . in javascript we use the `if` statement to perform tasks based on conditions.
+
+```js
+let  bestScore = 40;
+if (bestScore <= 45);{
+console.log(" please play again")
+}
+\\ prints: Please play again.
+```
+
+If the condition `(bestscore <= 45)` which means bestScore should be less than or equals to 45 to evaluates to `true`, Which means the code in the curly braces would run, but if it greater than 45 that means it will evaluate to `false`. Leaving the condition unevaluated. Therefore if the condition evaluates to `true` it would log `\\ Please play again` to the console.
+
+## The `if...else` Statement:
+
+The `if...else` statement is used when adding another condition in case to the statement. Meaning if the condition evaluates to false the other condition will run.
+
+```js
+let workingDays = 4;
+if (workingDays === 5) {
+    console.log('Thanks for your commitment');
+}else{
+    console.log('you were absent!');
+}
+\\ Prints: You were absent!.
+```
+
+## Logical Operators:
+
+Using conditional statements means using `true` or `false` and `boolean`.
+In javascript there are operators used with boolean known as logical operators. They add more logical expression to our conditions. We have three types of logical operators:
+
+- The and operator ( && ).
+- The or operator ( | | ).
+- The not operator also known as the bang
+  operator ( ! ).
+
+The `and` ( && ) operator: the `and` operator is used to check if the two conditions are `true`.
+
+```js
+if (weekDays === "sunday" && time === "12:00pm") {
+  console.log("it sunday afternoon");
+} else {
+  console.log("not aternoon yet");
+}
+```
+
+When using the `and` operator the statement only evaluate to `true` only when both of the conditions are `true`. If one of the condition evaluates to `false`, the second code block runs.
+
+The `or` ( | | ) operator: the `or` operator is use to check if one of the conditions is `true`.
+
+example:
+
+```js
+if (firstName === "basit" || firstname === "korede") {
+  console.log("Identified user");
+} else {
+  console.log("Unidentified user");
+}
+```
```

Comment:
Let's try to add the other when the code executes in the snippet?

_Please do this for all other occurrences._